### PR TITLE
Wallet unlock api

### PR DESCRIPTION
**Pull request by dwalintukan**

### CODE REVIEW

**Comment by dwalintukan** [comment]
Wait until this is merged:
https://github.com/bodhiproject/qweb3.js/pull/46

Then update qweb3 to 0.7.1.

File: package.json [hunk 1]
```diff
@@ -121,7 +121,7 @@
     "nedb-promise": "^2.0.1",
     "node-fetch": "^1.7.3",
     "pg": "^7.4.0",
-    "qweb3": "^0.6.0",
+    "qweb3": "^0.7.0",
```
Comment (frankobe) [inline package.json]: 0.7.1?

Comment (dwalintukan) [reply package.json]: was waiting for qweb3 pr to be merged in first. updated

**Comment by dwalintukan** [comment]
@frankobe this one good to go?

**Pull request merged**